server/routes/schedules.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const router = express.Router();
+const conn = require("../mariadb");
+const { StatusCodes } = require("http-status-codes");
+const {
+  getAllSchedules,
+  getScheduleById,
+  addSchedule,
+  updateSchedule,
+  deleteSchedule,
+} = require("../controller/schedulesController");
+
+router.use(express.json());
+
+router.get("/", getAllSchedules);
+router.get("/detail/:id", getScheduleById);
+router.post("/", addSchedule);
```

Comment:
서버측에서 DB에 insert하는 동작을 하는 함수를 표현할때는 add보다는 create라는 단어를 더 많이 쓰는 것 같아요!
```suggestion
router.post("/", createSchedule);
```